nixos/tests/ulogd.nix

```diff
@@ -1,9 +1,7 @@
 import ./make-test-python.nix ({ pkgs, lib, ... }: {
   name = "ulogd";
 
-  meta = with lib; {
-    maintainers = with maintainers; [ p-h ];
-  };
+  meta.maintainers = [ lib.maintainers.p-h ];
```

Comment:
that's one use case of `with`  that I think makes sense, the scope is small enough imo

Reply:
It is insane to write

```
meta = with lib; {
  maintainers = with maintainers; [ just-one-person ];
}
```

instead of

```
meta.maintainers = [ lib.maintainers.just-one-person ];
```

Further, these codes are old, and for years no one appeared saying "hey, I wanna help maintaining this module, I will add myself to maintainers list!"

There is no justification to "accomodate the code for future changes".

Effectively nothing is being lost.

Reply:
> It is insane to write `...` instead of `...`

No, it is not.

> There is no justification to "accomodate the code for future changes".

There is. There are no qualifications for whether a module is "old enough" to need to switch from `with maintainers;` to `[ maintainers.<name ];`.

Additionally, if your argument is that this module isn't getting more maintenance, what is the point of proactively making this change then?

Reply:
> No, it is not.

The semantical quirks of nested `with` are known enough.

> There is. There are no qualifications for whether a module is "old enough" to need to switch from with maintainers; to [ maintainers.<name> ];.

In the best case, it justifies only a single `with lib.maintainers`, not a ridiculous `with lib; with maintainers;`.

> what is the point of proactively making this change in the first place?

Uniformization.

Further, beginners (and veterans too) tend to look for existing code as template for future code.